test/common/extensions/filters/http/platform_bridge/platform_bridge_filter_test.cc

```diff
@@ -95,6 +126,10 @@ TEST_F(PlatformBridgeFilterTest, NullImplementation) {
   Http::TestResponseTrailerMapImpl response_trailers{{"x-test-trailer", "test trailer"}};
   EXPECT_EQ(Http::FilterTrailersStatus::Continue, filter_->encodeTrailers(response_trailers));
 
+  checkFilterState("NullImplementation", "0",
```

Comment:
Will add throughout the rest of the test suite once we are finalized on all the state we want to keep track of and dump.